package.json

```diff
@@ -21,8 +21,8 @@
   "main": "./dist/js/okta-sign-in.entry.js",
   "scripts": {
     "clean": "rimraf target && rimraf dist && rimraf build2",
-    "lint:report": "grunt lint --checkstyle && eslint -f checkstyle -o build2/OSW-eslint-checkstyle.xml . || true",
-    "lint": "grunt lint && eslint .",
+    "lint:report": "grunt lint && node ./buildtools/stylelint/stylelint.js checkstyle || true && eslint -f checkstyle -o build2/OSW-eslint-checkstyle.xml . || true",
```

Comment:
Is `|| true` needed for our stylelint script?

Reply:
Yepp, without that, if stylelint fails, we would not get to the eslint report